app/models/draft_complete_workflow.rb

```diff
@@ -1,37 +1,37 @@
 # frozen_string_literal: true
 
-# Simple draft / published state-based workflow: Start as draft (will be
-# visible, but not able to view the manifest.) When published, the
+# Simple draft / completed state-based workflow: Start as draft (will be
+# visible, but not able to view the manifest.) When completed, the
 # manifest is visible.
-class DraftPublishWorkflow < BaseWorkflow
+class DraftCompleteWorkflow < BaseWorkflow
   aasm do
     state :draft, initial: true
-    state :published
+    state :completed
 
     # ingest workflow
-    event :publish do
-      transitions from: :draft, to: :published
+    event :complete do
+      transitions from: :draft, to: :completed
     end
-    event :unpublish do
-      transitions from: :published, to: :draft
+    event :draft do
```

Comment:
Is this what the event is called in other workflows? It seems confusing to have an event and state with the same name.

Reply:
Understood, I can adjust this; I was basing it off of :https://github.com/pulibrary/figgy/blob/master/app/models/folder_workflow.rb#L11